py2pack/templates/opensuse.spec

```diff
@@ -78,11 +77,16 @@ BuildArch:      noarch
 %setup -q -n {{ name }}-%{version}
 
 %build
-{% if has_ext_modules %}export CFLAGS="%{optflags}"{% endif %}
-%python_build
+{% if has_ext_modules %}export CFLAGS="%{optflags}"
+{% endif %}%python_build
 
 %install
 %python_install
+{%- if has_ext_modules %}
+%python_expand %fdupes %{buildroot}%{$python_sitearch}
```

Comment:
Doesn't that BuildRequire fdupes? Also this is unrelated to the mentioned bug from the commit msg